src/main/java/frc/robot/Commands/IntakeCargo.java

```diff
@@ -26,7 +26,7 @@ public boolean isFinished() {
                                         return intakeToTransfer.getLimit();
                                     }
                                 },
-                                new MoveGenericSubsystem(transfer, Transfer.SPEED) {
+                                new MoveGenericSubsystem(transfer, Transfer.getInstance().getTransferSpeed()) {
```

Comment:
you already have the instance, why would you use `getInstance()` again and not the instance you already have?